engine/execution/computation/execution_verification_test.go

```diff
@@ -147,6 +147,7 @@ func executeBlockAndVerify(t *testing.T, txs [][]*flow.TransactionBody) *executi
 	fvmContext := fvm.NewContext(logger,
 		fvm.WithChain(chain),
 		//fvm.WithBlocks(blockFinder),
+		fvm.WithTransactionFeesEnabled(true),
```

Comment:
@m4ksio I added transaction fees to these tests. This make the first test (covering empty chunks) fail if the change in `module/chunks/chunkVerifier.go` is not applied. This is exactly because the system transaction fails during verification, but not during execution.

Applying the change in `module/chunks/chunkVerifier.go` makes the test pass again.